checks.d/openvpn.py

```diff
@@ -2,10 +2,27 @@
 import os
 import datetime
 from collections import defaultdict
+from collections import Counter
+import re
 
 # project
 from checks import AgentCheck
 
+IPV4_REGEX = re.compile('^((?:[0-9]{1,3}\.){3}[0-9]{1,3})(:\d+)*$')
+
+# Removes the port from an IP address
+# Currently assumes that the address is
+# an IPv4 address
+def strip_port(ipv4_address):
+    matches = IPV4_REGEX.match(ipv4_address)
+    if matches is not None:
+        return matches.group(1)
+    return None
+
+# Infers the /24 of the address.
+# e.g. 192.168.1.1 will return 192.168.1.0/24
+def infer_subnet_24(ipv4_address):
+    return '.'.join(ipv4_address.split('.', 3)[0:3] + ['0']) + '/24'
```

Comment:
Is the tag ok? Does it get converted to an `_` or anything? (Doesn't matter, just might be better to use it's final form like a tag pokemon.